spacy/tests/lang/nl/test_noun_chunks.py

```diff
@@ -0,0 +1,10 @@
+    doc = nl_tokenizer(txt)
+    with pytest.raises(ValueError):
+        list(doc.noun_chunks)
```

Comment:
Always very happy to see unit tests included with a new contribution :-) 

In addition to the "failing" test, it would be good to also have a positive test. You could define a document with tokens and parsing information, and the expected noun chunks, as has been done for Danish here: https://github.com/explosion/spaCy/blob/master/spacy/tests/lang/da/test_noun_chunks.py

Would you be able to add a few example cases for Dutch? We don't yet have these for all languages yet, but going forward I think it makes sense to include some more tests :-)

Reply:
Will do

Reply:
Also for this test, a short text (one sentence) is fine. There's no benefit to having a long text here.

Reply:
Ah, good point. Actually, this text isn't even correct Dutch most of the time, so just one correct Dutch sentence will suffice ;-)

Reply:
See recent commit, I have a short text with 4 sentences, along with a reference chunking.
There is one test to check that the ValueError exception is raised, and another test to check the chunking itself.